doc/api/HostDB.md

```diff
@@ -288,3 +402,46 @@ Examples
   ]
 }
 ```
+
+#### Hosts
+
+###### Request
+```
+/hostdb/hosts/ed25519:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef
```

Comment:
If I could go back in time, there would only ever be the hex/string version. The base64 version was never meant to be human readable, and it only slipped through because we didn't think that anyone would ever be directly looking at pubkeys.